src/components/screens/wallet/delegateProfile/delegateProfile.css

```diff
@@ -1,44 +1,102 @@
-.wrapper {
+.container {
   display: flex;
-  padding: 32px 16px;
+  flex-direction: column;
 }
 
-.delegateStats {
-  align-content: stretch;
-  box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
-  flex-wrap: wrap;
-  height: 60px;
-  list-style: none;
-  margin: 0;
-  padding: 0;
-  width: 100%;
-
-  & > li {
-    align-items: center;
-    color: var(--color-maastricht-blue);
-    display: flex;
-    font-family: var(--content-font);
-    font-size: var(--paragraph-font-size-l);
-    height: calc(100% / 2);
+.details {
+  & .itemContainer {
+    border-bottom: 1px solid #e1e3eb;
+    padding: 20px 0;
+    margin-left: 0;
+
+    &:last-child {
+      border: none;
+      padding-bottom: 0;
+    }
+
+    &:first-child {
+      padding-top: 0;
+    }
   }
 
-  & span.label {
-    color: var(--color-dark-gray);
-    font-size: var(--paragraph-font-size-s);
-    margin-right: 8px;
-    white-space: nowrap;
+  & .item {
+    margin-left: 16px;
+
+    & .value {
+      margin-top: 8px;
+    }
   }
 }
 
-@media (--medium-viewport) {
-  .delegateStats {
-    height: 90px;
+.performance {
+  padding: 0 !important;
+  display: flex;
+  flex-direction: row !important;
```

Comment:
If it is possible I think it would be better to not use !important. Can these styles be done in another way that does not require to use it?